hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml

```diff
@@ -468,6 +468,18 @@
                 </hz:query-caches>
             </hz:map>
 
+            <hz:map name="testMap5">
+                <hz:tiered-store enabled="true">
+                    <hz:in-memory-tier>
+                        <hz:capacity>128</hz:capacity>
+                    </hz:in-memory-tier>
+                    <hz:disk-tier enabled="true" capacity="108">
```

Comment:
why the in-memory tier represents capacity as a nested element while disk-tier has it as an attribute? Should we use the same style for both?

Reply:
in in-memory tier capacity has default value, but for the disk-tier capacity does not have a default, so it is an attribute and marked as required part of configuring disk-tier. We can make both nested elements, if we introduce a default to disk-tier capacity, shall we? 1 GB maybe?

Reply:
Yeah, let's make them similar as nested elements. About default I'm not sure. Maybe @ahmetmircik has idea? Maybe 10gb?

Reply:
or what if we remove `enabled` attribute from `disk-tier` and then set the default capacity to 0 which will mean `disk-tier` is not enabled. If the user wants `disk-tier` they will have to set the capacity to some positive value.

Reply:
@petrpleshachkov maybe we can have no upper limit as default.

Reply:
@ahmetmircik we discussed it towards the end of the standup, and decided to remove `capacity` from the `disk-tier`. In the future, once we add support for additional tiers, we can introduce it back.